docs/topics/maven.md

```diff
@@ -23,6 +23,27 @@ To use JDK 17, in your `.mvn/jvm.config` file, add:
 --add-opens=java.base/java.io=ALL-UNNAMED
 ```
 
+## Declare repositories
+
+To access artifacts in repositories that can be consumed by Maven builds, specify the ID and URL of each repository in
+the `<repositories>` tag:
+
+```xml
+<repositories>
+    <repository>
+        <id>mavenCentral</id>
+        <url>https://repo1.maven.org/maven2/</url>
+    </repository>
+</repositories>
+```
+
+> We recommend that you avoid adding `mavenLocal()` as a repository. If you declare `mavenLocal()` as a repository then you
```

Comment:
For Maven build tool using `mavenLocal()` is fine as it anyway downloads dependencies into it. Plus does not use Gradle metadata. I think this part should be removed from here. Or we need to clarify that this warning is for Gradle.

Reply:
I updated this warning to a note explaining that it is only something to worry about if you also use Gradle projects. Can you check it and let me know what you think?